components/atom/colorPanel/src/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const TYPES = {
+  DEFAULT: 'default',
+  CANVAS: 'canvas',
+  BASE: 'base',
+  CONTRAST: 'contrast',
+  CORPORATE: 'corporate',
+  HIGHLIGHT: 'highlight'
+}
+
+const getClassNames = ({type}) =>
+  classnames(
+    'sui-AtomColorPanel',
+    `sui-AtomColorPanel-${type}`
+  )
+
+const getStyle = ({opacity}) => ({
+  opacity
+})
+
+const AtomColorPanel = ({children, ...props}) => (
+  <div className={getClassNames(props)} style={getStyle(props)}>
```

Comment:
Is the opacity supposed to be for the component or for its background ?

Reply:
background, thanks!

Reply:
Then I request the change !! :-)